src/main/java/noppes/npcs/roles/JobFollower.java

```diff
@@ -15,6 +15,7 @@ public class JobFollower extends JobInterface{
 
 	public JobFollower(EntityNPCInterface npc) {
 		super(npc);
+		npc.stats.canDespawn = false;
```

Comment:
CNPCs already set this to false by default. Switching a role to follower shouldn't change this, it can already be directly changed in the "Respawn" GUI.

Reply:
I don't know why I did this. It doesn't solve the issue I was having, and is completely pointless as you pointed out. A while back I made NPC's that could spawn and despawn naturally but when I made them follow me they would eventually despawn anyway. 
What I wanted is for the npc to not despawn if it has an owner. So I guess I quickly did this without thinking and saw that none of the followers were despawning and didn't look into it further. Oops. 
I will actually setup the logic I want this time.